charts/cp-schema-registry/templates/schema-registry-ui.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  {{- if .Values.ui.annotations }}
+  annotations: {{ toYaml .Values.ui.annotations | nindent 4 }}
+  {{- end}}
+  labels:
+    app: {{ template "cp-schema-registry.fullname" . }}
+  {{- if .Values.ui.labels }}
+  {{ toYaml .Values.ui.labels | indent 4}}
+  {{- end}}
+  name: {{ template "cp-schema-registry.fullname" . }}-ui
+spec:
+  replicas: 1
```

Comment:
replicas is usually a good thing to put in values (to be customized, if required)

Reply:
Since it's fixed for the UI, I would like to leave it hardcoded, if you don't see any drawback.

Reply:
what do you mean "fixed for the UI" ?
the app can't work with (for example) 2 pods in // ?

Reply:
I mean that, AFAIK, there is no need to scale the ui

Reply:
In that case, I still think you should put into values, so that we can customize it one day.
It could be important for HA

Reply:
I will test if it works with more than one replica and then proceed to the change if it's ok

Reply:
Tested and working well, just pushed the change !

Reply:
Cool 👍